README.md

```diff
@@ -452,6 +479,38 @@ export default MyComponent = () => {
 
 ## Migrating between versions
 
+### Migrating from 4.x to 5.x
+
+From version 5.0, the Security component explicitly requires prop [restoreOriginalUri](#restoreoriginaluri). Example of implementation of this callback for `react-router`:
```

Comment:
it would be great to also explain why Security comp requires this prop, like `to decouple from react-router`.